src/assets/stylesheets/index.scss

```diff
@@ -438,12 +438,12 @@ a.btn {
 }
 
 ::selection {
-  background: var(--bs-secondary);
-  color: #fff;
+  background: rgba(255, 204, 0, 0.761);
+  color: #333;
 }
 ::-moz-selection {
-  background: var(--bs-secondary);
-  color: #fff;
+  background: rgba(255, 204, 0, 0.761);
+  color: #333;
 }
```

Comment:
* [ ] FYI Firefox supports `::selection` so this could go

https://codepen.io/sconway/pen/arZMqe